webapp/src/components/post_type_jitsi/post_type_jitsi.tsx

```diff
@@ -139,6 +175,18 @@ export class PostTypeJitsi extends React.PureComponent<Props, State> {
             </div>
         );
     }
+
+    render() {
+        return (
+            <IntlProvider
```

Comment:
I used my own IntlProvider because the the original mattermost translations machinery wasn't working, so I did all by my own in my side.